src/PlaneWaveBasis.jl

```diff
@@ -438,20 +433,24 @@ In-place version of `G_to_r`.
 @timing_seq function G_to_r!(f_real::AbstractArray3, basis::PlaneWaveBasis,
                              f_fourier::AbstractArray3)
     mul!(f_real, basis.opIFFT, f_fourier)
+    f_real .*= basis.G_to_r_normalization
```

Comment:
Use 5-argument version of `mul!` ?

Reply:
Not sure FFTW supports it, better to be more explicit here

Reply:
I just thought if they will eventually then we don't have to think about it, but I'm pretty sure right now they don't.